server/repository/src/db_diesel/changelog/test.rs

```diff
@@ -461,7 +468,7 @@ async fn test_changelog_name_and_store_id_in_trigger() {
         |connection, r| {
             InvoiceLineRowRepository::new(connection)
                 .delete(&r.id)
-                .unwrap()
+                .unwrap();
```

Comment:
These `;` are needed as otherwise would be returning `change_log_id` which isn't handled.